Mathlib/Algebra/Order/Field/Power.lean

```diff
@@ -203,7 +206,8 @@ def evalZPow : PositivityExt where eval {u α} zα pα e := do
     | _ => throwError "not a ^ n where n is a literal or a negated literal"
   orElse result do
     let ra ← core zα pα a
-    let ofNonneg (pa : Q(0 ≤ $a)) (_oα : Q(LinearOrderedSemifield $α)) :
+    let ofNonneg (pa : Q(0 ≤ $a))
+        (_oα : Q(Semifield $α)) (_oα : Q(LinearOrder $α)) (_oα : Q(IsStrictOrderedRing $α)) :
```

Comment:
Is it OK here and in other places to use the same variable names?

Reply:
Do we even need variable names here and elsewhere?